menpo/io/output/base.py

```diff
@@ -113,6 +113,8 @@ def export_pickle(obj, fp, overwrite=False):
         The string path or file-like object to save the object at/into.
     overwrite : `bool`, optional
         Whether or not to overwrite a file if it already exists.
+    protocol : `int`, optional
+        Save with a different pickle protocol the file. Mostly for python 3.
```

Comment:
It would be good to add some kind of table here in the doc that explains what each protocol int value means.